### PR TITLE
Added extkingpin module

### PR DESCRIPTION
**Pull request by saswatamcode**
This PR adds in the `extkingpin` module which provides the `PathOrContent` flag with environment variable substitution.

### CODE REVIEW

**Comment by bwplotka** [comment]
Can you run `go mod tidy` (see lint)?

**Comment by saswatamcode** [comment]
Sure! I did run it before raising PR but I think I set wrong go version(1.16).

File: README.md [hunk 1]
```diff
@@ -207,3 +205,25 @@ or via [bingo](https://github.com/bwplotka/bingo) if want to pin it:
 go install github.com/bwplotka/bingo
 bingo get -u github.com/efficientgo/tools/copyright
 ```
+
+### Module `github.com/efficientgo/tools/extflag`
```
Comment (bwplotka) [inline README.md:209]:
```suggestion
### Module `github.com/efficientgo/tools/extkingpin
```

Comment (bwplotka) [reply README.md:209]: WDYT?

File: README.md [hunk 2]
```diff
@@ -207,3 +205,25 @@ or via [bingo](https://github.com/bwplotka/bingo) if want to pin it:
...
+
+This module provides the PathOrContent flag type which defines two flags to fetch bytes. Either from file (\*-file flag) or content (\* flag). Also returns the content of a YAML file with substituted environment variables.
+
+```go mdox-gen-exec="sh -c 'tail -n +6 extflag/doc.go'"
```
Comment (bwplotka) [inline README.md:213]: 💪🏽 How great is that! (:

**Review by bwplotka — APPROVED** [review]
Solid, proposed one idea, otherwise LGTM!

File: README.md (hunk 2 above)
Comment (saswatamcode) [reply README.md:213]: 😃

File: README.md (hunk 1 above)
Comment (saswatamcode) [reply README.md:209]: Sure!

**Review by bwplotka — APPROVED** [review]
Thanks! LGTM 💪🏽

**Pull request merged**